ansible/jenkins.yml

```
@@ -0,0 +1,135 @@
+      enabled: yes
+
+  - name: Install Trivy
+    shell: |
+      curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/main/contrib/install.sh | sh -s -- -b /usr/local/bin
+    args:
+      creates: /usr/local/bin/trivy
+
+  - name: Install kind (Kubernetes in Docker)
+    when: ansible_architecture == "x86_64"
+    block:
+      - name: Download kind binary
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)